k8s-handle.py

```diff
@@ -19,9 +19,9 @@
 log = logging.getLogger(__name__)
 
 if 'LOG_LEVEL' in sys.argv:
-    logging.basicConfig(level=sys.argv['LOG_LEVEL'])
+    logging.basicConfig(level=sys.argv['LOG_LEVEL'], format=settings.LOG_FORMAT, datefmt=settings.LOG_DATE_FORMAT)
 else:
-    logging.basicConfig(level='INFO')
+    logging.basicConfig(level='INFO', format=settings.LOG_FORMAT, datefmt=settings.LOG_DATE_FORMAT)
```

Comment:
Can we not use if/else for `LOG_LEVEL` ?

Reply:
I want some:
```
log = logging.getLogger(__name__)

logging.basicConfig(level=settings.LOG_LEVEL, format=settings.LOG_FORMAT, datefmt=settings.LOG_DATE_FORMAT)
```

without if/else logic .

Reply:
Ok, i got it. It just old code from time when settings.py never exists. So we can just leave one line:
```python
logging.basicConfig(level=settings.LOG_LEVEL, format=settings.LOG_FORMAT, datefmt=settings.LOG_DATE_FORMAT)
```